### DISCUSSION TITLE
Possible to make a remap depending on the current system keyboard layout?

### DISCUSSION THREAD

**Discussion by reserved-word** [cajhin/capsicain#97] [Q&A]
I use two kb layouts, I switch between them using `Alt+Shift`. I would like to have an independent set of remapable keys for each of them. Perhaps this can be implemented with two independent configs, but I did not find in the documentation a way to bind the config to the current active system layout.

Is this possible?

**Comment by cajhin** [comment]
not sure if I understand the question. What does the 'current active system layout' have to do with it? I guess you need different special keys depending on which win layout you use.

First, capsicain knows nothing about how Windows processes the keypresses, afterwards. A config for Ctrl-Z works the same, no matter if Windows later makes a Ctrl-Y out of it.

The straight solution to (example) map physical keys Ctrl-Z to either 'Delete' or to Backspace, is to have two seperate configs like

```
[CONFIG_1]   #make Delete
COMBO Z [..&.] > key(DEL)
[CONFIG_2]   #make Backspace
COMBO Z [..&.] > key(BSP)
```

Then you have to manually switch Capsicain configs every time you switch Win kb layout.

A way to automate this, I would have to try myself if/how this works, would be to catch Alt+Shift (or another combo you choose), then switch config AND send out Alt+Shift to Windows.

If I misunderstood, please clarify. An example of what exactly you want to achieve is always helpful.
